### ISSUE TITLE
Random stuttering in Final Fantasy I & II: Dawn of Souls

### ISSUE THREAD

**Issue by kurayami6** [mgba-emu/mgba#1356]
Sometimes when walking around the overworld the game will randomly start stuttering very slightly. Turning sync to audio off and back on seems to fix it. Windows 7, i5, GTX 970. All settings default except for audio buffer which is set to 512 because the default was causing it's own constant stuttering.

**Comment by endrift** [comment]
What is your sound card? Try upgrading your drivers.

**Comment by kurayami6** [comment]
Audio drivers are already all up to date. Here's my computer specs. If the problem is just that my hardware is shit then this issue can be closed. 

![Capture](https://user-images.githubusercontent.com/31968468/54451058-0c6b1b80-47a6-11e9-966c-78e8fac36ec3.PNG)

**Comment by endrift** [comment]
Realtek is generally low quality but a lot of people have it so I should try to support it.

**Comment by endrift** [comment]
Just to check, do you have sync to video enabled too?

**Comment by kurayami6** [comment]
No, just sync to audio. If I turn that off the audio sounds like ear piercing garbage.
